go/iot/configurePi.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)"
+
+echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
```

Comment:
You should be able to download a secret key JSON and authorize with it.

We dont need the GCP SDK on the pi.

Let's merge this as it is for now, and I will show you how to use the secret keys.